docs/README.md

```diff
@@ -2,35 +2,55 @@
 
 <a href="https://www.curseforge.com/minecraft/mc-mods/no-chat-reports"><img alt="curseforge" height="28" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@2/assets/compact/available/curseforge_vector.svg"></a> <a href="https://modrinth.com/mod/no-chat-reports"><img alt="modrinth" height="28" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@2/assets/compact/available/modrinth_vector.svg"></a> <a href="https://github.com/Aizistral-Studios/No-Chat-Reports"><img alt="github" height="28" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@2/assets/compact/available/github_vector.svg"></a> <a href="https://gitlab.com/Aizistral-Studios/No-Chat-Reports"><img alt="gitlab" height="28" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@2/assets/compact/available/gitlab_vector.svg"></a>
 
-This mod strips cryptographic signatures that since 1.19 are attached to every message sent in the chat. Removing them makes it impossible to track and associate your chat messages with your Minecraft and, by extension, Microsoft account.
+This mod strips cryptographic signatures which are attached to every chat message sent from 1.19 and onwards. Removing them makes it impossible to track and associate your chat messages with your Minecraft client, and, by extension, Microsoft account.
 
-**As of 1.19.1, it also disables Player Chat Reporting.**
+**Disables Player Chat Reporting from 1.19.1 and onwards.**
 
-Please notice that, while I am aware of [Guardian](https://github.com/nodusclient/guardian) and certain other exploits that allow you to counteract reporting system on client against the will of the server, I am consciously choosing to not integrate them as part of this mod. No Chat Reports will remain effective on client side only if given server allows it to be. If you do not like servers that enforce chat signing and fully support chat reporting - I advise to simply not play on them.
+**Also removes signatures on 1.19 as a security measure, even though the ability to report players only got added in 1.19.1.**
```

Comment:
Now that I look at it, it seems redundant as line 5 already says the same. And the changes of line 7 are then unnecessary as well.

Reply:
No, I meant keeping line 7 but not changing it.

Reply:
Basically reverting that section to the state it was in before my PR?

Reply:
Yes.